### ISSUE TITLE
[Feature] Backup PCs as soon as possible (e.g. WakeupSchedule every minute)

### ISSUE THREAD

**Issue by hamster65** [backuppc/backuppc#246]
Especially when the backuppc-server is not running all the time, it would be nice to start backups as soon as possible and not e.g. only at the full hour, as defined per WakeupSchedule. With WakeupSchedule , this is hard to archive because you would have to enter hundreds of wakeup times.
My WakeupSchedule already looks like this (which is too long for the text field in the webinterface, by the way...) and still I often have to wait until backups start.
20, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 21, 22, 23, 1.3, 2.3, 3.3, 4.3, 5.3, 6.3, 7.3, 8.3, 9.3, 10.3, 11.3, 12.3, 13.3, 14.3, 15.3, 16.3, 17.3, 18.3, 19.3, 20.3, 21.3, 22.3, 23.3, 1.6, 2.6, 3.6, 4.6, 5.6, 6.6, 7.6, 8.6, 9.6, 10.6, 11.6, 12.6, 13.6, 14.6, 15.6, 16.6, 17.6, 18.6, 19.6, 20.6, 21.6, 22.6, 23.6

**Comment by hobbes1069** [comment]
Alternative idea...

What about if a host goes too long without a backup it goes on a separate list of hosts that get pinged at more frequent intervals and if successful, a new backup is initiated.

**Comment by hamster65** [comment]
Good idea. That way, hosts that have not been backuped for a long time would also be placed in front of the backup queue because the separate list would be checked more often.

Another idea would be to support time ranges in WakeupSchedule, like 8-22, and then have a second option to configure what interval to use, eg. 5 minutes.

At least for my environments (both home and work), WakeupSchedule is not that usefull because I want the PCs to be backuped as soon as possible. The exact time does not matter at all.

**Comment by GWHAYWOOD** [comment]
Would it not make more sense to use the existing facility which can trigger a backup from the command line?

You could for example put something in a cron job to suit your circumstances.

Closing for now as this is very old, but please get back to me if I have misunderstood.
